oak_client/src/transport.rs

```diff
@@ -0,0 +1,60 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+pub trait Transport {
```

Comment:
can we use https://github.com/project-oak/oak/blob/c583847bc49da459b00e438eabc90004603d3f84/micro_rpc/src/lib.rs#L44-L49 instead?

Reply:
Thanks
Will look into it